cabal-install/cabal-install.cabal

```diff
@@ -66,6 +66,7 @@ library
     exposed-modules:
         -- this modules are moved from Cabal
         -- they are needed for as long until cabal-install moves to parsec parser
+        -- TODO #6101 should probably be deleted too
```

Comment:
this comment I can't quite understand. It'd be good to make it a bit more detailed.

Reply:
Ah yes, it was more a comment for me currently :D 
it is about the modules below, I guess we should delete them:
```
        Distribution.Deprecated.ParseUtils
        Distribution.Deprecated.ReadP
        Distribution.Deprecated.ViewAsFieldDescr
```
They are part of the legacy parser, but I do not know if we can just delete them or if they are also used in other places than the ProjectConfig parser. So I need to investigate it further

Reply:
AH, gotcha, thanks!